internal/pkg/deploy/cloudformation/stack/scheduled_job.go

```diff
@@ -68,6 +68,30 @@ const (
 	every = "@every "
 )
 
+type errScheduleInvalid struct {
+	reason string
```

Comment:
Was there a reason why we didn't use instead? not that import it's just unusual
```suggestion
	reason error
```